pkg/controller/perconaservermongodb/mgo.go

```diff
@@ -319,17 +311,23 @@ func (r *ReconcilePerconaServerMongoDB) getConfigMemberForPod(ctx context.Contex
 	}
 
 	switch pod.Labels[naming.LabelKubernetesComponent] {
-	case "arbiter":
+	case naming.ComponentArbiter:
 		member.ArbiterOnly = true
 		member.Priority = 0
-	case "mongod", "cfg":
+	case naming.ComponentMongod, "cfg":
 		member.Tags = tags
-	case "nonVoting":
+	case naming.ComponentNonVoting:
 		member.Tags = util.MapMerge(mongo.ReplsetTags{
-			"nonVoting": "true",
+			naming.ComponentNonVoting: "true",
 		}, tags)
 		member.Priority = 0
 		member.Votes = 0
+	case naming.ComponentHidden:
+		member.Tags = util.MapMerge(mongo.ReplsetTags{
+			naming.ComponentHidden: "true",
+		}, tags)
+		member.Priority = 0
```

Comment:
From what I understand reading the documentation, the only configuration needed to setup a hidden replica set contains these 2 properties: 1) priority 0 and 2) the hidden tag equal to true. Correct?

https://www.mongodb.com/docs/manual/tutorial/configure-a-hidden-replica-set-member/#configuration-procedure

Reply:
correct